sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,63 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
@pedrobaeza Usually we put the license image on top of the file you just need to swap it with the title

https://github.com/pedrobaeza/sale-workflow/blob/8.0-sale_order_recalculate_prices/sale_order_price_recalculation/README.rst